app/javascript/src/utilities/utils.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import React from 'react'
+
+function CSRFToken ({win = window}: {win?: Window}) {
```

Comment:
`utils` is a rather generic name that could eventually become a huge file. That's why I gather utils functions in different files `json-utils`and `createReactWrapper`. For consistency we should do the same here... or move everything into this one.

Reply:
Also it should be a `jsx` I think.

Reply:
It's good to start with just one file. When it gets bigger, then refactor. This is the same with having shitloads of tiny components spread... doesn't make sense. We go one file until it gets big enough and we need separation of concerns.

Reply:
The eternal dispute hehe